services/weightscale.md

```diff
@@ -0,0 +1,43 @@
+# Weight Scale
+
+    identifier: 0x1f4d5040
+    extends: _sensor
+
+A weight measuring sensor.
+
+## Registers
+
+    ro weight: u16.16 g @ reading
+
+The reported weight.
+
+    ro weight_error?: u16.16 g @ reading_error
+
+The estimate error on the reported reading.
+
+    rw zero_offset: u16.16 g 0x80
+    
+Calibrated zero offset error on the scale, i.e. the measured weight when nothing is on the scale.
+
+    rw gain: u16.16g 0x81
+    
+Calibrated gain on the weight scale error.
```

Comment:
Shouldn't this be some sort of multiplier?

Reply:
Yes internally it's factor applied to the weight, offset + factor * measured_weight

Reply:
The I think this should be unit-less, if it's a multiplier.

Also good to make these things optional - I imagine some chips may just store that stuff internally and not give you access.